src/lib/BufferedDuplex.ts

```diff
@@ -62,11 +80,15 @@ export class BufferedDuplex extends Duplex {
 		})
 	}
 
-	_read(size?: number): void {
+	public override _read(size?: number): void {
```

Comment:
The `override` keyword is mandatory here to indicate we are overriding the behaviour of a method that was already defined in the parent class. This improves maintainability.